frontend/src/reducers/auth.js

```diff
@@ -40,7 +40,7 @@ export default function (state = initialState, action) {
     case AUTH_ERROR:
     case LOGIN_FAIL:
       // eslint-disable-next-line no-undef
-      localStorage.removeItem('token');
+      // localStorage.removeItem('token');
```

Comment:
Why commented out? If not needed you can remove this right?

Reply:
Yup, agreed !! 👍